Readme.md

```diff
@@ -0,0 +1,29 @@
+# DigitalBank
+Digital Bank is an example online digital banking application for exploratory purposes.
+## Getting Started
+* Clone [this repository](https://github.com/digisic/digitalbank-gen-one)
+## Prerequisites
+* Install [Java SDK](https://openjdk.java.net/)
+* Install [Node.js](https://nodejs.org/download/release/v12.21.0/)
+* Install [Visual Studio Code](https://code.visualstudio.com/download)
+## Executing VSCode
+* Create a new folder and add it to the workspace
+* In a new terminal run: npm init 
+* Install WebdriverIO: npm install webdriverio --save-dev
+* Install the client: npm install @wdio/cli
+* Create the configuration file: npx wdio config
+* Install Chai: npm install chai --save-dev
+* Install Chai-WebdriverIO: npm install chai-webdriverio --save-dev
+## Executing Tests
+* Change the file wdio.conf.js: specs './tests/**/*.js'
+* Add in package.json: "test": "wdio ./wdio.conf.js"
+* JUnit Tests - Execute "npm run test"
```

Comment:
JUnit is a testing library for java. In this case, you are using the wdio test executor (it actually uses mocha under the hood, but it's fine if you just refer to it as wdio's executor)

Reply:
done